src/en/help-google.md

```diff
@@ -0,0 +1,114 @@
+Title: Help with Google Compute Engine clouds
+
+# Using the Google Compute Engine public cloud
+
+Juju already has knowledge of the GCE cloud, so unlike previous versions there
+is no need to provide a specific configuration for it, it 'just works'. GCE
+will appear in the list of known clouds when you issue the command:
+  
+```bash
+juju list-clouds
+```
+And you can see more specific information (e.g. the supported regions) by 
+running:
+  
+```bash
+juju show-cloud google
+```
+
+If at any point you believe Juju's information is out of date (e.g. Amazon just 
```

Comment:
we might want to change this blurb on all our sheets. how can a user determine if something is out of date? might be better to just say "run this to make sure it is up to date".